test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js

```diff
@@ -52,7 +52,7 @@ const backYourStackCollectiveData = {
 
 describe('server/graphql/v2/mutation/CreateCollectiveMutations', () => {
   beforeEach('reset db', async () => {
-    await utils.resetTestDB();
+    await utils.resetTestDB({ groupedTruncate: false });
```

Comment:
Why these `groupedTruncate` here in tests?

Reply:
I randomly had a deadlock there. Should remove, unrelated.